setup.py

```diff
@@ -4,12 +4,16 @@
 import setuptools
 from setuptools import find_packages, setup
 
+
+_THIS_DIR = os.path.dirname(os.path.realpath(__file__))
+
+PYTHON_REQUIRES = ">=3.7"
```

Comment:
I almost missed updating the `python_requires` kwarg in the `setup()` call below. Moving this up here as a constant next to the other related trove classifiers, so that we hopefully won't forget about it next time.